Framework/DataHandling/src/LoadMask.cpp

```diff
@@ -179,14 +404,11 @@ void LoadMask::initDetectors() {
  *                   in masking
  *   @param tomask:  true to mask, false to unmask
  *   @param singledetids: list of individual det ids to mask
- *   @param pairdetids_low: list of lower bound of det ids to mask
- *   @param pairdetids_up: list of upper bound of det ids to mask
+
  */
 void LoadMask::processMaskOnDetectors(const detid2index_map &indexmap,
                                       bool tomask,
-                                      std::vector<int32_t> singledetids,
-                                      std::vector<int32_t> pairdetids_low,
-                                      std::vector<int32_t> pairdetids_up) {
+                                      std::vector<int32_t> singledetids) {
```

Comment:
I might be blind to the bigger picture here, but why not pass `singledetids` as reference?